### ISSUE TITLE
[eslint-plugin] `valid-shorthands` mishandles `oklch` colors

### ISSUE THREAD

**Issue by dwei-figma** [facebook/stylex#1231]
### Describe the issue

The `valid-shorthands` rule misreports `borderColor` authored as a `oklch` value as a disallowed shorthands usage. The rule also suggests a bad auto-fix breaking the `oklch` expression by spaces.

### Expected behavior

For the rule to not complain in this case.

### Steps to reproduce

```ts
const styles = stylex.create({
  foo: {
    // eslint-disable-next-line @stylexjs/valid-shorthands
    borderColor: 'oklch(0.928 0.006 264.531)',
  },
})
```

**Comment by mellyeliu** [comment]
Similar bug was reported here a while back that we never got around to fixing: https://github.com/facebook/stylex/issues/981 

We use an external package (https://www.npmjs.com/package/css-shorthand-expand) that may be handling this split wrong. We've had some issues with it before, we'll have to patch over this or migrate to something better

**Comment by dwrth** [comment]
As a test I did a quick fork of this package and simply added a regex for `oklch` colors, this seems to fix the issue. It also seems pretty easy to add more as well https://github.com/dwrth/css-shorthand-expand

_Edit: It did in fact not work 🥲_

**Comment by cnaples79** [comment]
I submitted a PR for this.
